spec/teaspoon/instrumentation_spec.rb

```diff
@@ -116,6 +116,20 @@ def ignore(paths)
 
         expect(subject.add?(response, "QUERY_STRING" => "instrument=true")).to be(false)
       end
+      
+      context 'with default ignored files' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.